build.gradle

```diff
@@ -40,10 +40,12 @@ buildscript {
 }
 
 plugins {
```

Comment:
Let's not add this in the whole repo. Let's add in the sub modules where it's missing.

Reply:
But it is the same for every submodules, so is it make more sense to at it to the whole repo

Reply:
@ylwu-amzn do you have any suggestion here?

Reply:
I'm ok for this as this will be applied to all modules. But feel free to comment if you have concerns